pkgs/development/libraries/polkit/default.nix

```diff
@@ -108,10 +104,12 @@ stdenv.mkDerivation rec {
     ]))
   ];
 
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR= "${placeholder "out"}/lib/systemd/system";
```

Comment:
https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/environment-variables-go-to-env.md

Reply:
Ah! I saw some packages using env but some didn't and I wasn't sure which one was preferred, thanks for the link :)